src/readers/F3DReaderBoilerPlate.h.in

```diff
@@ -0,0 +1,90 @@
+/**
+ * This file is auto-generated by CMake, do not edit directly.
+ */
+
+#ifndef F3D@F3D_READER_NAME@_h
+#define F3D@F3D_READER_NAME@_h
+
+#include "F3DReaderFactory.h"
+
+#if @F3D_READER_HAS_GEOMETRY_READER@
+#include <@F3D_READER_VTK_READER@.h>
+#endif
+
+#if @F3D_READER_HAS_SCENE_READER@
+#include <@F3D_READER_VTK_IMPORTER@.h>
+#endif
+
+#include <vtkVersion.h>
+#include <vtksys/SystemTools.hxx>
+
+class F3D@F3D_READER_NAME@ : public F3DReader
+{
+public:
+  F3D@F3D_READER_NAME@() = default;
+
+  /*
+   * Get the name of this reader
+   */
+  const std::string GetName() const override { return "@F3D_READER_NAME@"; }
+
+  /*
+   * Get the short description of this reader
+   */
+  const std::string GetShortDescription() const override { return "@F3D_READER_DESCRIPTION@"; }
+
+  /*
+   * Get the extensions supported by this reader
+   */
+  const std::vector<std::string> GetExtensions() const override
+  {
+    static const std::vector<std::string> ext = { @F3D_READER_EXTENSIONS@ };
+    return ext;
+  }
+
+  /*
+   * Get the mimetypes supported by this reader
+   */
+  virtual const std::vector<std::string> GetMimeTypes() const override
+  {
+    static const std::vector<std::string> types = { @F3D_READER_MIMETYPES@ };
```

Comment:
do we actually need this one ?

Reply:
Not necessarily, but it can be useful later maybe? If you prefer, I'll remove it, let me know.

Reply:
lets keep it, maybe we can add a f3d --info or at the end of --help ?

Reply:
actually, add it to --reader-list

Reply:
done